### DISCUSSION TITLE
Namespaces file behavior

### DISCUSSION THREAD

**Discussion by cvolante54** [ory/keto#668] [Q&A] [answered]
Seems to me like when using a file to configure the namespaces (ex. namespaces: file///...), this will only allow for[ one namespace to be created](https://github.com/ory/keto/blob/master/internal/driver/config/namespace_watcher.go#L152), should that be the expected behavior or should an array of namespaces be read from the file?

**Comment by vinckr** [comment]
I think this is expected behavior at the moment, see this issue: https://github.com/ory/keto/issues/613. 
But @zepatrik can probably clarify this!

**Comment by zepatrik** [comment, marked as answer]
We already resolved this in #669, somehow missed this discussion.
